bin/saya/src/args/data_availability.rs

```diff
@@ -44,6 +44,28 @@ pub struct CelestiaOptions {
     #[arg(requires = "celestia_node_url")]
     pub celestia_namespace: Option<String>,
 }
+#[derive(Debug, Args, Clone)]
+pub struct StarknetAccountCliInput {
```

Comment:
Is that ok to move to something like `StarknetSettlementOptions` or `StarknetAccountOptions`?

Reply:
done